third-party/nvapi-opensource/readme.txt

```diff
@@ -0,0 +1,3 @@
+Opensource version of nvapi
+Updates can be found at https://download.nvidia.com/XFree86/nvapi-open-source-sdk/

```

Comment:
Okay, I see it doesn't really seem to be available in any updated git repo (that I could find anyway). Could we just download and extract this with CMakeLists? I'm not really a fan of including other source code directly if it can be avoided.

https://cmake.org/cmake/help/latest/command/file.html#transfer
https://cmake.org/cmake/help/latest/command/file.html#archive-extract

Reply:
But let's look upon this objectively

What cons is this going to bring?
- We'll be effectively breaking repo history once they change the url, and it's going to happen sooner or later
- It can cause immense annoyance in places with spotty internet (hey, I like taking my laptop to the mountains)

What pros?
- I don't see any

Reply:
It's just, I think we're spending time trying to solve a problem that wasn't a problem to begin with. And if I start looking into that `archive-extract`, it will introduce actual problems that will require actual solving and more back and forth communication.

Reply:
If this acts as the regular nvapi drs then it's not even a real library at all, it's just a loader for the real library inside the driver.
It's made to be forward compatible forever, it's not something to keep updated.
The Nvidia drs dev guide is from 2011 and points this out.

Reply:
That's the thing, the files in question are not code, but constants. And we don't really need to update them unless we would want to change some newly added parameters. Which we won't be needing for years, most likely ever. And if we can't find the parameter in question, that's the sure sign to update. Which takes 3 minutes since it's trivial and kept in one place. You will spend order of magnitudes more time on automation.